README.md

```diff
@@ -131,53 +131,55 @@ controller_server:
       plugin: "mppi::Controller"
       time_steps: 15
       model_dt: 0.1
-      batch_size: 400
+      batch_size: 300
       vx_std: 0.2
       vy_std: 0.2
       wz_std: 1.0
       vx_max: 0.5
       vy_max: 0.5
       wz_max: 1.3
-      iteration_count: 2
-      prune_distance: 1.5
+      iteration_count: 1
+      prune_distance: 1.2
       transform_tolerance: 0.1
       temperature: 0.25
       motion_model: "DiffDrive"
       visualize: false
-      critics: ["ReferenceTrajectoryCritic", "GoalCritic", "GoalAngleCritic", "ObstaclesCritic", "PreferForwardCritic" ]
-      ReferenceTrajectoryCritic:
-        reference_cost_power: 1
-        reference_cost_weight: 3.0
-        enable_nearest_goal_critic: true
-        nearest_goal_offset: 2
-        nearest_goal_count: 2
-        nearest_goal_cost_power: 1
-        nearset_goal_cost_weight: 1.0
-        enable_nearest_path_angle_critic: true
-        nearest_path_angle_offset: 6
-        nearest_path_angle_cost_power: 1
-        nearest_path_angle_cost_weight: 1.0
+      critics: ["ReferenceTrajectoryCritic", "LocalGoalCritic", "GoalCritic", "GoalAngleCritic", "ObstaclesCritic" ]
       GoalCritic:
         goal_cost_power: 1
-        goal_cost_weight: 5.0
+        goal_cost_weight: 3.0
       GoalAngleCritic:
         goal_angle_cost_power: 1
-        goal_angle_cost_weight: 5.0
+        goal_angle_cost_weight: 3.0
         threshold_to_consider_goal_angle: 0.20
       ObstaclesCritic:
         consider_footprint: true
-        collision_cost: 2000.0
+        collision_cost: 100.0
```

Comment:
What benefit did reducing the cost to collision have? I would think outrageously large is the right answer